cypress/support/index.js

```diff
@@ -17,6 +17,9 @@
 import './commands';
 import '@cypress/code-coverage/support';
 
+Cypress.on('uncaught:exception', (err, runnable) => {
+	return false
+});
```

Comment:
why is this required?

Reply:
I am trying to check for the error message which the system throws, so when doing that I am getting an exception, to handle that I have added the above.